src/main/java/com/hitesh/TicTacToe/commands/CommandExecutor.java

```diff
@@ -0,0 +1,12 @@
+package com.hitesh.TicTacToe.commands;
+
+import com.hitesh.TicTacToe.model.Command;
+import com.hitesh.TicTacToe.model.Game;
+
+public abstract class CommandExecutor {
```

Comment:
why abstract class over interface, given you aren't defining a basic implementation of any method? https://stackoverflow.com/questions/5835311/what-is-the-advantage-of-interface-over-abstract-classes